src/common/plugin.ts

```diff
@@ -47,10 +55,54 @@ export abstract class Plugin implements IPlugin {
 
     const response = this.container.channelService.hasPermission(channelName, this.permission);
     if (!response) {
-      message.reply(`Please use this command in a \`${this.permission}\` channel`);
+      const rooms = Object.values(Constants.Channels[this.permission]).splice(0, 3);
+      let addonText = '';
+
+      // not sure if this will cause issues as I do pop id, please let me know
+      const chanInfo: CChannelInfo[] = rooms
+        .filter((room) => {
+          return this.container.guildService
+            .getChannel(room)
+            .permissionsFor(message.member || '')
+            ?.has('VIEW_CHANNEL');
+        })
+        .map((room) => new CChannelInfo(this.container.guildService.getChannel(room).id, room))
+        .splice(0, 2);
+
+      try {
+        if (this.permission.toString() === 'Private') {
+          addonText = ` This is primarily the class channels, and any channels we haven't defined.`;
+        } else if (rooms.length === 0) {
```

Comment:
`rooms.length` is not the same as `chanInfo.length`

Reply:
This is so the ending to the message changes accordingly.  After all we can't display all the channels else public would be a wall of text,  so I decided 2 displayed channels will be the cut-off point.